test_proposal.json

```diff
@@ -0,0 +1,12 @@
+{
+  "messages": [
+    {
+      "@type": "/craft.exp.v1beta1.MsgJoinDaoByIbcAsset",
+      "join_address": "craft1hj5fveer5cjtn4wd6wstzugjfdxzl0xp86p9fl",
+      "gov_address": "craft10d07y265gmmuvt4z0w9aw880jnsr700jm5qjn0",
+      "amount": "1000000"
+    }
+  ],
+  "metadata": "4pIMOgIGx1vZGU=", 
+  "deposit": "100000000000stake"
```

Comment:
Can you explain this? - 
If you own EXP you are apart of the DAO already. EXP (stake) is not transferable.

Reply:
Oh, this only for test bash. I mean 'stake' here is transable denom like 'ucraft'.